specification/synapse/data-plane/Microsoft.Synapse/stable/2021-04-01/symsAPI.json

```diff
@@ -27,7 +27,7 @@
     "/databases/ExecuteChange": {
       "post": {
         "summary": "Batch execution of DDL Payload.",
-        "operationId": "ExecuteChange",
+        "operationId": "SyMS_ExecuteChange",
```

Comment:
What's the meaning of "SyMS"? If possible, could we use a more meaningful name?

Reply:
SyMS stands for **SY**napse **M**etadata **S**ervice. Would you prefer SynapseMetadataService instead? Let me know your thoughts.

Reply:
Just personal thought, I think `SynapseMetadata` is enough.
Add other SDK owner for awareness: @weidongxu-microsoft @pshao25 @qiaozha

Reply:
Done. Changed to SynapseMetadata

Reply:
Please merge if everything looks good.

cc: @weidongxu-microsoft @lmazuel  @pshao25  @qiaozha